lib/Dialect/Calyx/CalyxOps.cpp

```diff
@@ -724,17 +722,17 @@ void ParOp::getCanonicalizationPatterns(RewritePatternSet &patterns,
 //===----------------------------------------------------------------------===//
 // WiresOp
 //===----------------------------------------------------------------------===//
-static LogicalResult verifyWiresOp(WiresOp wires) {
-  auto component = wires->getParentOfType<ComponentOp>();
+LogicalResult WiresOp::verify() {
+  auto component = (*this)->getParentOfType<ComponentOp>();
   auto control = component.getControlOp();
 
   // Verify each group is referenced in the control section.
-  for (auto &&op : *wires.getBody()) {
+  for (auto &&op : *this->getBody()) {
     if (!isa<GroupInterface>(op))
       continue;
     auto group = cast<GroupInterface>(op);
     auto groupName = group.symName();
-    if (SymbolTable::symbolKnownUseEmpty(groupName, control))
+    if (::SymbolTable::symbolKnownUseEmpty(groupName, control))
```

Comment:
Why is the full qualification needed here?

Reply:
It will raise a compiler error.
```
../lib/Dialect/Calyx/CalyxOps.cpp:735:22: error: no member named 'symbolKnownUseEmpty' in 'mlir::OpTrait::SymbolTable<circt::calyx::WiresOp>'
    if (SymbolTable::symbolKnownUseEmpty(groupName, control))
```
can be better edited like `::mlir::SymbolTable`, just like it in the function `ComponentOp::print` , `ComponentOp::parse` .